src/components/LoginDialog.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <q-card style="min-width: 25%">
+    <q-card-section>
+      <div class="text-h6">
+        Log in
+      </div>
+    </q-card-section>
+    <form @submit.prevent="onSubmit">
+      <q-card-section>
+        <q-input
+          v-model="username"
+          dense
+          outlined
+          placeholder="Username"
+          stack-label="Stack Label"
```

Comment:
```suggestion
          :stack-label="true"
```
This attribute is looking for a boolean. The colon (short for v-bind) allows javascript expressions like the `true` object.

Also, you don't have a `label` attribute, so this isn't doing anything at this point.

Reply:
I apologize, I did a copy paste from the quasar doc and didn't double-check